webofneeds/won-bot/README.md

```diff
@@ -53,46 +108,37 @@ new ActionOnFirstNEventsListener(
 bus.subscribe(ActEvent.class, atomCreator);
 ```
 
-## The EventListenerContext
-A bot is connected to the resources it uses through the [EventListenerContext](src/main/java/won/bot/framework/eventbot/EventListenerContext.java). Resources of interest might be:
-* The [LinkedDataSource](/webofneeds/won-core/src/main/java/won/protocol/util/linkeddata/LinkedDataSource.java) obtained via `EventListenerContext.getLinkedDataSource()`. It is used to query linked data.
-* The [AtomProducer](src/main/java/won/bot/framework/component/atomproducer/AtomProducer.java) obtained via `EventListenerContext.getAtomProducer()`. It's a facility for creating atoms, much like an iterator. Depending on the configured implementation, data can be read from any source and transformed into an atom. To see how this is done in practice, look at the [MailFileAtomProducer](src/main/java/won/bot/framework/component/atomproducer/impl/MailFileAtomProducer.java)
-* The [WonMessageSender](/webofneeds/won-core/src/main/java/won/protocol/message/sender/WonMessageSender.java) obtained via `EventListenerContext.getWonMessageSender()`. This object allows the bot to send WoN messages.
-
-## The BotContext
-A bot manages the data it needs to operate in the [BotContext](src/main/java/won/bot/framework/bot/context/BotContext.java). Most importantly, the botContext contains all atom URIs that the bot is responsible for. That particular information is required by the framework to decide which events should be routed to the bot, as it might be managing a number of bots. Other information in the bot context is just for the bot's internal purposes (i.e. your purposes).
-
-## The BotContextWrapper
-A bot always contains a specific [BotContextWrapper](src/main/java/won/bot/framework/bot/context/BotContextWrapper.java). This class or the respective subclasses enable you to access the BotContext described above, and facilitates HelperMethods for BotContext-DataRetrieval
-## The Bot's App
-The java application that runs a bot is usually a spring-boot application that loads all the necessary config and then starts the bot. For examples, look at the existing [Bot Apps](src/main/java/won/bot/app). 
+### Sending WoN messages
 
-To create a new app (which is what you want if you want to run a bot), it's easiest to copy it's app class and the corresponding spring config files, and then making modifications.
+For doing anything on the WoN, a bot must send messages to a node or to specific atoms. This is done in two steps:
```

Comment:
Maybe it's better not to contract Web of Needs here? It felt a bit confusing to me to read "on the WoN"